modules/nf-core/bascule/environment.yml

```diff
@@ -0,0 +1,14 @@
+name: nf-core_bascule
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - bioconda::r-bascule=1.0.1
+  - conda-forge::compilers=1.9.0
+  - conda-forge::python=3.8.20
+  - conda-forge::pip=24.2
+  - conda-forge::r-ggplot2=3.5.2
+  - conda-forge::r-reticulate=1.42.0
+  - conda-forge::r-tidyverse=2.0.0
```

Comment:
You include the tidyverse, which is pretty big. Do you actually need the entire tidyverse, or can you pick only the pckages you need and get a smaller image?